Module installation in GNU / Linux

I am in KDE Neon distribution based on Ubuntu 18.04. When I follow the installation steps: Clone repository and `jmvtools::install()`

I get the following error:
```
jmvtools::install()

jamovi compiler

jamovi found at /usr/bin/flatpak
wrote: glmdif.h.R
wrote: glmdif.src.js
wrote: glmdif.js
wrote: ordinaldif.h.R
wrote: ordinaldif.src.js
wrote: ordinaldif.js
wrote: ordinalreliability.h.R
wrote: ordinalreliability.u.yaml
wrote: ordinalreliability.src.js
wrote: ordinalreliability.js
wrote: rdttest.h.R
wrote: rdttest.u.yaml
wrote: rdttest.src.js
wrote: rdttest.js
wrote: testroc.h.R
wrote: testroc.src.js
wrote: testroc.js
wrote: ttestcor.h.R
wrote: ttestcor.u.yaml
wrote: ttestcor.src.js
wrote: ttestcor.js
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module './glmDIF.events' from '/home/brian/git/jamoviPsychoPDA/build/js'
    at /home/brian/R/x86_64-pc-linux-gnu-library/3.6/jmvtools/node_modules/jamovi-compiler/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/home/brian/R/x86_64-pc-linux-gnu-library/3.6/jmvtools/node_modules/jamovi-compiler/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/brian/R/x86_64-pc-linux-gnu-library/3.6/jmvtools/node_modules/jamovi-compiler/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /home/brian/R/x86_64-pc-linux-gnu-library/3.6/jmvtools/node_modules/jamovi-compiler/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:123:15)
```

--- a/jamovi/glmdif.u.yaml
+++ b/jamovi/glmdif.u.yaml
@@ -4,7 +4,7 @@ jus: '2.0'
 stage: 0
 compilerMode: tame
 events:
-  update: './glmDIF.events::update'
+  update: './glmdif.events::update'
 children:
   - type: VariableSupplier
     name: itemSupplier
@@ -19,7 +19,7 @@ children:
             name: item
             isTarget: true
             events:
-              change: './glmDIF.events::onChange_item'
+              change: './glmdif.events::onChange_item'
       - type: TargetLayoutBox
         label: Anchor Items
         children:
@@ -171,7 +171,7 @@ children:
         persistentItems: false
         stretchFactor: 1
         events:
-          update: './glmDIF.events::onUpdate_plotVarsSupplier'
+          update: './glmdif.events::onUpdate_plotVarsSupplier'
         children:
           - type: TargetLayoutBox
             children:

